test/test-Worker.js

```diff
@@ -194,6 +195,9 @@ test('Worker emits "died" if worker process died unexpectedly', async () =>
 
 test('worker process ignores PIPE, HUP, ALRM, USR1 and USR2 signals', async () =>
 {
+	if (os.platform() === 'win32') 
```

Comment:
Please, add a comment here explaining why such a `return` in case of Windows.

Reply:
yep. already a comment added.